src/api/Config/types.ts

```diff
@@ -252,6 +252,7 @@ export interface ILeafletConfig {
 export interface IAccessConfig {
   floatingLicenses?: number;
   authRequired?: boolean;
+  localAuth: boolean;
```

Comment:
Same comment as the server PR.
I have a preference for negative by default options it is more convenient for migrations.

```suggestion
  disableLocalAuth?: boolean;
```